create a get started landing page (#5269)
fishtown-analytics/docs.getdbt.com: website/docs/docs/introduction.md

dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.

Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint). [Join the dbt Community](https://www.getdbt.com/community/join-the-community) to learn how other data practitioners around the world are using dbt, share your own experiences, and get help with your dbt projects.

> <Lightbox src="/img/docs/cloud-overview.jpg" width="60%" title="dbt works alongside your ingestion, visualization, and other data tools, so you can transform data directly in your cloud data platform." />

Reviewer: It might be beneficial for the description here to be a sentence placed above the photo (as a lead in) and the `title` here changed to describe what is in the photo.

## dbt products

> You can use dbt to quickly and collaboratively transform data and deploy analytics code following software engineering best practices like version control, modularity, portability, CI/CD, and documentation. This means anyone on the data team can safely contribute to production-grade data pipelines.

Reviewer: ```suggestion
Use dbt to quickly and collaboratively transform data and deploy analytics code following software engineering best practices like version control, modularity, portability, CI/CD, and documentation. This means anyone on the data team can safely contribute to production-grade data pipelines.
```

<!-- removing per Leona's feedback and pending review
To cater to the diverse needs of data teams, dbt offers two primary solutions:

- [**dbt Cloud**](#dbt-cloud) &mdash; A comprehensive, full managed service enabling teams to develop, test, deploy, and explore data products safely and reliably.
- [**dbt Core**](#dbt-core) &mdash; An open-source CLI tool that's suitable for users who prefer a manual setup.
-->

### dbt Cloud

> dbt Cloud offers the fastest, most reliable and scalable way to deploy dbt. Develop, test, schedule, and investigate data models using a single, fully managed service through a web-based user interface (UI).

Reviewer: ```suggestion
dbt Cloud offers the fastest, most reliable, and scalable way to deploy dbt. Develop, test, schedule, and investigate data models using a single, fully managed service through a web-based user interface (UI).
```

Exclusive features of dbt Cloud include:

> - Hosted environment so it’s faster to get up and running,  without having to manage ongoing infrastructure maintenance

Reviewer: ```suggestion
- Hosted environments so it’s faster to get up and running without having to manage ongoing infrastructure maintenance
```

- Development experiences tailored to multiple personas (in-browser [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or locally with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation))
- Features designed to help you deploy and scale with ease, such as: [stateful metadata](/docs/dbt-cloud-apis/discovery-api), [intuitive data discovery with dbt Explorer](/docs/collaborate/explore-projects), [in-app job scheduler](/docs/deploy/deployments), out-of-the-box [CI/CD workflows](/docs/deploy/ci-jobs), [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro), and more.

You can learn about plans and pricing on [www.getdbt.com](https://www.getdbt.com/pricing/)

Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features) and try one of the [dbt Cloud quickstarts](/docs/get-started-dbt).

### dbt Core

> [dbt Core](/docs/core/about-core-setup) is an open-source tool that enables data teams to transform data and suitable for users who prefer manual setup. You can install dbt Core through the command line.

Reviewer: Same feedback about adding a couple of bullet points. Maybe
- Broader support for adapters from a variety of vendors and community contributors.
- Provides individuals or small teams the flexibility to run their own infrastructure end-to-end.
- The open-source nature allows for deeper customization to meet specific needs.

Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint).

<Lightbox src="/img/docs/cloud-overview.jpg" width="60%" title="dbt works alongside your ingestion, visualization, and other data tools, so you can transform data directly in your cloud data platform." />

## dbt products

You can use dbt to quickly and collaboratively transform data and deploy analytics code following software engineering best practices like version control, modularity, portability, CI/CD, and documentation. This means anyone on the data team can safely contribute to production-grade data pipelines.

To cater to the diverse needs of data teams, dbt offers two primary solutions:

- [**dbt Cloud**](#dbt-cloud) &mdash; A comprehensive, full managed service enabling teams to develop, test, deploy, and explore data products safely and reliably.

> - [**dbt Core**](#dbt-core) &mdash; An open-source CLI tool that enables data teams to transform data using analytics engineering best practices.

Reviewer 2: I wonder if we should use "suitable for users who prefer manual setup" here like you do on the GS page. I love that framing because it's super helpful

> - [**dbt Core**](#dbt-core) &mdash; An open-source CLI tool that enables data teams to transform data using analytics engineering best practices.

Author: ```suggestion
- [**dbt Core**](#dbt-core) &mdash; An open-source CLI tool that's suitable for users who prefer a manual setup.
```

> Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint).

Author: i wonder if we should include anything about the dbt community on this page?

Author: added a sentence
